### ISSUE TITLE
Unable to build conda on 1.0.0-DEV.42

### ISSUE THREAD

**Issue by kafisatz** [JuliaPy/Conda.jl#112]
See here, line 514
https://travis-ci.org/kafisatz/DecisionTrees.jl/jobs/412500252

**Comment by aitzkora** [comment]
I make a PR #114  about this problem, but i'm not sure I respected the spirit of isdefined, must be reviewed

**Comment by brilhana** [comment]
On `1.0`:
```julia
(v1.0) pkg> build Conda
  Building Conda → `~/.julia/packages/Conda/a196m/deps/build.log`
┌ Error: Error building `Conda`: 
│ ERROR: LoadError: ArgumentError: isdefined: too few arguments (expected 2)
│ Stacktrace:
│  [1] top-level scope at none:0
│  [2] include at ./boot.jl:317 [inlined]
│  [3] include_relative(::Module, ::String) at ./loading.jl:1038
│  [4] include(::Module, ::String) at ./sysimg.jl:29
│  [5] include(::String) at ./client.jl:388
│  [6] top-level scope at none:0
│ in expression starting at /Users/alexandrebrilhante/.julia/packages/Conda/a196m/deps/build.jl:7
└ @ Pkg.Operations /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:1068
```

**Comment by stevengj** [comment]
Closed by #113
